packages/ui/src/utils/get-user-updated-properties-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { isDefined } from './is-defined';
+
+export function getUserUpdatedPropertiesSchema(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj1: Record<string, any>,
```

Comment:
I think you could use `KaotoSchemaDefinition['schema']` instead of `Record<string, any>` for the schema properties, for the input model, you could use `Record<string, unknown>`